ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -176,6 +169,46 @@ export default class AlertTable extends React.Component {
     return { failureStatus };
   };
 
+  selectAlertsByStatus = (selectedStatus) => {
+    const { alertSummary } = this.state;
+    let { allSelected } = this.state;
+
+    const status = {
+      all: 'all',
+      none: 'none',
+      triaged: 'triaged',
+      untriaged: 'untriaged',
+    };
+    let selectedAlerts = [...alertSummary.alerts];
+
+    if (selectedStatus === status.none) {
+      selectedAlerts = [];
+      allSelected = false;
+    }
+
+    if (selectedStatus === status.all) {
```

Comment:
Could we do an if/else if chain? It's easier to assert the code's implications this way.
(For example, if the status is none, we'd hit the 1st branch & the 3rd one.)